app/src/sharedTest/java/org/oppia/android/app/topic/conceptcard/ConceptCardFragmentTest.kt

```diff
@@ -420,23 +422,25 @@ class ConceptCardFragmentTest {
   fun testConceptCardFragment_severalConceptCards_dismissAll() {
     launchTestActivity().use { scenario ->
       scenario.onActivity { activity ->
-        val fragmentSkill0 = ConceptCardFragment.bringToFrontOrCreateIfNew(
+        ConceptCardFragment.bringToFrontOrCreateIfNew(
           TEST_SKILL_ID_0,
           ProfileId.getDefaultInstance(),
           activity.supportFragmentManager
         )
-        assertThat(fragmentSkill0).isNotNull()
+        val fragmentSkill0 =
+          activity.supportFragmentManager.fragments.filterIsInstance<ConceptCardFragment>().single()
+        ConceptCardFragment.bringToFrontOrCreateIfNew(
+          TEST_SKILL_ID_1,
+          ProfileId.getDefaultInstance(),
+          activity.supportFragmentManager
+        )
         val fragmentSkill1 =
-          ConceptCardFragment.bringToFrontOrCreateIfNew(
-            TEST_SKILL_ID_1,
-            ProfileId.getDefaultInstance(),
-            activity.supportFragmentManager
-          )
-        assertThat(fragmentSkill1).isNotNull()
-        fragmentSkill0!!.showNow(activity.supportFragmentManager, fragmentSkill0.tag)
-        assertThat(activity.supportFragmentManager.fragments.size).isEqualTo(2)
+          activity.supportFragmentManager.fragments.filterIsInstance<ConceptCardFragment>().single()
+        assertThat(fragmentSkill1).isNotEqualTo(fragmentSkill0)
+        fragmentSkill0.showNow(activity.supportFragmentManager, fragmentSkill0.tag)
+        assertThat(activity.supportFragmentManager.fragments).hasSize(2)
         ConceptCardFragment.dismissAll(activity.supportFragmentManager)
-        assertThat(activity.supportFragmentManager.fragments.size).isEqualTo(0)
+        assertThat(activity.supportFragmentManager.fragments).hasSize(0)
```

Comment:
```suggestion
        assertThat(activity.supportFragmentManager.fragments).isEmpty()
```

Small nit: ``isEmpty()`` is the more idiomatic method for this case.

Reply:
Done